src/context/theme.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {
+  createContext,
+  FunctionComponent,
+  useCallback,
+  useContext,
+  useState,
+} from "react";
+import {
+  CustomTheme,
+  govWalletTheme,
+  sallyTheme,
+} from "../common/styles/themes";
+
+interface ThemeContext {
+  theme: CustomTheme;
+  setTheme: (themeName: string | undefined) => void;
+}
+
+export const ThemeContext = createContext<ThemeContext>({
+  theme: sallyTheme,
+  setTheme: () => {}, // eslint-disable-line @typescript-eslint/no-empty-function
```

Comment:
I saw a few other examples of this being:

```
setTheme: () => undefined

setTheme: () => void
```

Is there a difference between having an empty function and the above examples?

Reply:
Hmm dont think there's a difference, good observation on this! Perhaps we should standardise the entire code in a subsequent PR?